### DISCUSSION TITLE
Easily Switch Between Dark and Light mode

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#598] [Ideas]
It would be a big quality of life improvement if users were able to switch between light and dark mode with a toggle or a keyboard shortcut. Or be able to schedule dark mode to turn on at certain times, like after 9:00pm everyday, and then turn off at 8:00am. As of right now, I can't find any way to do this. If there is a method, please let me know!

**Comment by rcalixte** [comment]
There was an applet published recently specifically for this purpose. [Automatic dark/light themes](https://cinnamon-spices.linuxmint.com/applets/view/397) by @guillaume-mueller.

It does not currently support a keyboard shortcut but if you open a Feature Request (the "Issues" button in the link above) and notify the author, the conversation could at least be started on the topic. This also applies to the scheduling aspect.
